### ISSUE TITLE
inline: ['remote'] is not working when multiple files are processed

### ISSUE THREAD

**Issue by evromalarkey** [jakubpawlowicz/clean-css#966]
https://github.com/jakubpawlowicz/clean-css#how-to-optimize-multiple-files and https://github.com/jakubpawlowicz/clean-css#how-to-process-remote-imports-correctly does not work together, it throws this error:
```
C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\optimizer\level-1\optimize.js:628
    switch (token[0]) {
                 ^

TypeError: Cannot read property '0' of undefined
    at level1Optimize (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\optimizer\level-1\o
ptimize.js:628:18)
    at optimize (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\clean.js:128:5)
    at C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\clean.js:104:29
    at C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\read-sources.js:26:64
    at loadOriginalSources (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\load-or
iginal-sources.js:26:5)
    at C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\read-sources.js:26:14
    at applySourceMaps (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\apply-sourc
e-maps.js:34:5)
    at Object.callback (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\read-source
s.js:25:12)
    at doInlineImports (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\read-source
s.js:200:25)
    at inlineLocalStylesheet (C:\Users\Administrator\Desktop\test\node_modules\clean-css\lib\reader\read-
sources.js:327:10)
```

### Environment

* clean-css version - `npm ls clean-css`: 4.1.7
* node.js version - `node -v`: 6.11.2
* operating system: Windows 10

### Configuration options

```js
var CleanCSS = require('clean-css');
new CleanCSS({
  inline: ['all']
}).minify({
  'path/to/file/one': {
    styles: '@import url(https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css);'
  },
  'path/to/file/two': {
    styles: '@import url(https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css);'
  }
}, function (error, output) {
  console.log(output.styles);
})
```

**Comment by jakubpawlowicz** [comment]
I can see it too. There seems to be a problem when mixing local and remote imports. Fix is coming!

**Comment by jakubpawlowicz** [comment]
This is fixed in 4.1.8.
